### PR TITLE
add code owners and CONTRIBUTING

### PR DESCRIPTION
**Pull request by feerrenrut**
### Link to issue number:
None

### Summary of the issue:
With code owners we can automatically request a code review from a code owner when a PR is no longer a draft.
Hopefully this helps contributors communicate they are ready for a review from core developers. 
To facilitate this, new contributions should start in draft state, and be converted to "ready for review" when initial checks are complete.

### Description of how this pull request fixes the issue:
- Adds a code owners file, docs: https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners
- Docs for auto-assign code reviewer: https://docs.github.com/en/organizations/organizing-members-into-teams/managing-code-review-assignment-for-your-team
- Adds a CONTRIBUTING file which points to our full guide: https://github.com/nvaccess/nvda/wiki/Contributing

### Testing strategy:
When this PR is marked as no longer a draft, a core dev should be automatically requested for review.

### Known issues with pull request:
- Will contributors remember to start PR's as a draft. Is there a way to enforce that?

### Change log entry:
None

### Code Review Checklist:

This checklist is a reminder of things commonly forgotten in a new PR.
Authors, please do a self-review and confirm you have considered the following items.
Mark items you have considered by checking them.
You can do this when editing the Pull request description with an x: `[ ]` becomes `[x]`.
You can also check the checkboxes after the PR is created.

- [x] Pull Request description is up to date.
- [x] Unit tests.
- [x] System (end to end) tests.
- [x] Manual tests.
- [x] User Documentation.
- [x] Change log entry.
- [x] Context sensitive help for GUI changes.

### CODE REVIEW

**Comment by CyrilleB79** [comment]
Hello
Would it be worth to specify explicitly in the initial comment of the PR template (PULL_REQUEST_TEMPLATE.md) that the PR should be opened as a draft and set as ready when it can be reviewed?
Or specify that the wiki has been updated in April 2021?
Just some thoughts...

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+# subdirectories.
+# /docs/ @doctocat
+
+# Start of NVDA config
+
+# By default auto request review from NV Access developer team.
+* @nvaccess/developers
+
+# For changes to the userGuidem auto request review from userDocs team
+/user_docs/en/user_guide.t2t @nvaccess/userDocs
```
Comment (michaelDCurran) [inline .github/CODEOWNERS:46]: Typo:
```suggestion
/user_docs/en/userGuide.t2t @nvaccess/userDocs
```

Comment (feerrenrut) [reply .github/CODEOWNERS:46]: oops good catch.

**Comment by christopherpross** [comment]
Hey,
I just want to make a quick note that we could force contributors to draft pr by using a bot.
This bot could close pull requests that were not marked as draft, with the note that only draft pr's are accepted. I would think of other useful features but only marginally. Or we could build a checkbox into the PR template that the user has to tick when the PR is ready. In this case, the bot would then add or mention or otherwise notify the code owners as reviewers.
Just a few ideas.

**Comment by feerrenrut** [comment]
Thanks for the suggestion @christopherpross. I think there are cases where it is valid to skip the draft step, but for most PR's there is at least an initial period where feedback is given on the overall approach / UX, it would be good if we only went to detailed PR after that. 

For now we can continue as is, and see if it actually causes a problem.

**Review by michaelDCurran — APPROVED** [review]

**Pull request merged**